### PR TITLE
Style system

### PR DESCRIPTION
**Pull request by Koyamot**
added finale banner

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,8 @@
       }}();
     </script>
     <meta charset="utf-8" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (drew-ross) [inline public/index.html:11]: Let's move these imports into the SCSS instead.

**Review by drew-ross — CHANGES_REQUESTED** [review]
Looks good but I'd like to see a few changes and stylesheet cleanups before merging:
- Put the new font import into index.scss instead of the index.html (replace the old one there)
- Move gameContainer.scss file into the /pages directory
- Remove the style imports in Lobby.tsx, import the SCSS file in index.scss under //Page Component Stylesheets
- Remove the style imports in Pregame.tsx, import the SCSS file in index.scss under //Page Component Stylesheets
- Change the Lobby.scss and Pregame.scss filenames to camel-case to be consistent

File: src/styles/index.scss [hunk 2]
```diff
@@ -7,14 +7,18 @@
 
 // Google Font Imports
 @import url("https://fonts.googleapis.com/css2?family=Bangers&family=Oswald:wght@100;200;300;400;500;600;700;800;900&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Luckiest+Guy&display=swap');
 
-// Page Component Stylesheets
```
Comment (drew-ross) [inline src/styles/index.scss:11]: Keep this comment in

File: src/styles/index.scss [hunk 3]
```diff
@@ -7,14 +7,18 @@
...
 @import "./components/pages/landingPage.scss";
 @import "./components/pages/finale.scss";
+@import "./components/pages/lobby.scss";
+@import "./components//pages/pregame.scss";
+
```
Comment (drew-ross) [inline src/styles/index.scss:16]: Remove the extra line

File: src/styles/index.scss [hunk 4]
```diff
@@ -7,14 +7,18 @@
 
 // Google Font Imports
 @import url("https://fonts.googleapis.com/css2?family=Bangers&family=Oswald:wght@100;200;300;400;500;600;700;800;900&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Luckiest+Guy&display=swap');
 
-// Page Component Stylesheets
 @import "./components/pages/landingPage.scss";
 @import "./components/pages/finale.scss";
+@import "./components/pages/lobby.scss";
+@import "./components//pages/pregame.scss";
```
Comment (drew-ross) [inline src/styles/index.scss:15]: There are two "/" in "//pages".

**Review by drew-ross — CHANGES_REQUESTED** [review]
Looks really good Kyla, just a few more small changes please. Thanks!

**Comment by Koyamot** [comment]
Edits complete.

**Review by drew-ross — APPROVED** [review]

**Pull request merged**